pkg/crc/machine/hyperkit/constants.go

```diff
@@ -6,7 +6,7 @@ import "fmt"
 
 const (
 	MachineDriverCommand = "crc-driver-hyperkit"
-	MachineDriverVersion = "0.12.9"
+	MachineDriverVersion = "0.12.10-beta1"
```

Comment:
I see you put a built binary in https://github.com/code-ready/machine-driver-hyperkit/releases/tag/v0.12.10-beta1
Fwiw, setting the version number in the `machine-driver-hyperkit` repo to 0.12.10, and changing it here might have been enough, people could build it from git, and then copy it to the right lo cation.